### DISCUSSION TITLE
Tuya WSD500A (Temperature & humidity sensor) Jitter

### DISCUSSION THREAD

**Discussion by hannpet** [Koenkk/zigbee2mqtt#19912] [Devices]
I recently added a Tuya WSD500A Sensor to my Z2M-Network.

Aside from a hughe Temp-Offset (which I don't mind and already fixed it), I'm finding that the sensor has a enormous Jitter, as shown in the added Screenshot.

Is there anything that can be done via the configuration, or is this maybe simply a defective device?

I'd be very interested if this is a rare exception or if many devices act like this.
Thanks!

<img width="509" alt="Bildschirmfoto 2023-11-29 um 11 30 06" src="https://github.com/Koenkk/zigbee2mqtt/assets/2811798/c43f2ebc-834f-430c-8ee9-0cb7fdce348d">

**Comment by tomhanax** [comment]
Same here, I have two of them and both behave the same. Probably you can not expect much for that price.
